test.js

```diff
@@ -21,3 +21,25 @@ test('emojis over text', function (t) {
   t.is(e.toEmoji('heart'), '❤️')
   t.is(e.toShortCode('❤️'), 'heart')
 })
+
+test('custom shortcodes to emojis', function (t) {
+  t.is(e.toEmoji('keet_laughs'), '😂')
+  t.is(e.toEmoji('keet_party'), '🥳')
+  t.is(e.toEmoji('keet_love'), '😍')
+  t.is(e.toEmoji('keet_music'), '😌🎶')
+  t.is(e.toEmoji('keet_bitcoin'), 'bitcoin')
+  t.is(e.toEmoji('keet_tether'), 'tether')
+  t.is(e.toEmoji('keet_pear'), '🍐')
+  t.is(e.toEmoji('holepunch'), '🕳️🥊')
+
+  t.is(e.toShortCode('😂'), 'joy')
+  t.is(e.toShortCode('🥳'), 'keet_party') // should be 'hooray'
```

Comment:
@mafintosh can you review please?
I added a test and it shows that 🥳 is converting to `keet_party` instead of `hooray`. 
Should it be fixed or I can keep it `as is`?

If it should be fixed I can suggest extend `toShortCode` and take nearest pointers around 'mid' and see its icons. If icon is the same as target icon, then I can compare shortcodes and select a shortcode that is not started with 'keet'.

Reply:
@mafintosh also not sure about  `t.is(e.toShortCode('bitcoin'), 'keet_bitcoin')`
I added prefix `keet_` to bitcoin because there is no 'bitcoin' emoji in emojis set. But in future it can be added I think. And to avoid conflicts I put a `keet_` prefix.
But may be we will have a weird situation if we convert 'bitcoin price is ...' text to text with emojis. It will bee `keet_bitcoin price is ...` I think. So, there is no 100% backward compatibility. Probably we shouldnt add emoji for `bitcoin` to emojis-index.

Reply:
i think to fix that you just wanna add them to data before adding the others.

one thing i am realising now though is that this is a public module, so we should prop move the "compat" list to just be hardcoded here or be a .json file here